README.md

```diff
@@ -75,11 +75,14 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
 5. Apache configuration
 
 If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:  (run by user who has root privileges)
-```bash
+    
+    ```bash
     sudo a2enmod rewrite
     sudo a2dissite default
     sudo a2ensite $projectname
     sudo service apache2 reload
+    ```
```

Comment:
These lines all need to be aligned all the way to the left and then it should work

Reply:
got it.  done.